tests/garage/envs/test_envs.py

```diff
@@ -43,28 +43,43 @@
     MountainCarEnv,
 ]
 
+mujoco_classes = [
+    SwimmerMazeEnv,
+    PointMazeEnv,
+    AntMazeEnv,
+]
```

Comment:
I think it's time to split out test_envs into individual tests. Can you check out @gonzaiva's PR and coordinate with him on how to merge yours and his?

Reply:
Alternatively you can just merge and let him rebase, since this is relatively small. But please make sure he integrates your new tests in his change (it's pretty big so it would be easy for him to miss).

Reply:
I think rebasing this will be easier as it's smaller.